.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```

Comment:
Are you intending to add the the .idea/* files to this commit? What benefits does it add?

Reply:
No.